version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "5.0.2-SNAPSHOT"
+ThisBuild / version := "5.0.2-SNAPSHOT"
```

Comment:
Please set version `5.1.0` cos new release approach requires the version to be in place